### PR TITLE
Initial guides framework

### PR DESCRIPTION
**Pull request by sghung**
The initial guides framework with a sample guide.

### CODE REVIEW

File: docs/_includes/topbar.html [hunk 1]
```diff
@@ -16,10 +16,14 @@
                     <a class="nav-link cw-nav-link cw-header-link-text" href="news.html">News</a>
                 </li>
                 
-                <li class="nav-item cw-navbar-item cw-header-link-news">
+                <li class="nav-item cw-navbar-item cw-header-link-blog">
```
Comment (sishida) [inline docs/_includes/topbar.html:19]: Will the "blog" tab replace the "news" tab? I thought we wanted both "blogs" and "news" as separate tabs.

**Comment by sishida** [comment]
@sghung Thanks for all your work on this! I noticed that `author_sample.png` looks like it's an IBM logo? Shouldn't we use a Codewind logo instead? Let me know if you need an image file for it. Thanks!

**Comment by sishida** [comment]
@MelHopper, @j-c-berger This PR will help the "Guides" tab appear on the Codewind.dev website. Please take a look and add a comment if there's anything you'd like to change!

@sghung I'm also thinking it would be good to get a developer who has HTML/CSS/Jekyll experience to add their review, too. :)

Thanks, all!

**Comment by j-c-berger** [comment]
Hello all, thank you for your efforts on this!

I have a PR #545 for Codewind in VS Code and PR #544 for Codewind in Eclipse. I think both are ready to be merged, but would like a review first. Since this is our first time merging PRs for quick guides, I'm not sure whom to ask for a review. @MelHopper, I would assume you, but maybe anyone from ID can suffice, so @micgibso or @sishida? 

To make sure I'm understanding it correctly, once the PRs are ready to be merged, does Steven merge them as 1 PR so it's all part of 1 delivery? 

Thanks!

**Comment by sghung** [comment]
@j-c-berger for your PRs, they need to be in the _guides folder. If that is the case, then you can merge first and then I merge my PR after (I would remove the sample md file for guides so that doesn't show up)

**Comment by j-c-berger** [comment]
@sghung, thanks and we'll get mine in there first then :)

File: docs/css/guides.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.guides-post-paragraph {
+    text-align: justify;
+    overflow: hidden;
+    margin-top: 23px;
+    margin-bottom: 35px;
+}
+
+.guides-post-author-name {
```
Comment (sishida) [inline docs/css/guides.css:74]: I think we can omit author info. Unlike blogs, we don't need to specify authors of guides.

File: docs/css/guides.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        width: 100%;
+        padding-top: 46px;
+    }
+
+    @media (max-width: 767.98px) {
+        padding-top: 24px;
+    }
+}
+
+.guides-post-author-image {
```
Comment (sishida) [inline docs/css/guides.css:28]: I think we can omit author images. If we need an image here, maybe we could use the Codewind logo.

**Pull request merged**